index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>InnovationX Africa Summit</title>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"/>
+        <link rel="stylesheet" href="./style/style.css">     
+    </head>
+    <body>
+      <section class="container">
+        <header>
+          <div class="top-navigation">
+            <div class="media-links">
+              <ul>
+                <li><a><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+                <li><a><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                <li><a>English</a></li>
+                <li><a>My Page</a></li>
+                <li><a>Logout</a></li>
+              </ul>
+            </div>
+            <div>
+              <li class="logo-link"><a href="index.html"><img class="summitlogo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
```

Comment:
- Hey, when using the li, it should however be wrapped in the ul tag as the ul tag is used to group the list of unordered list items. Currently, you've used it incorrectly here. Since you're using it just for the purpose of wrapping your logo in a container, you can use the div tag here, as it's appropriate and semantically correct. please, let's fix this.
- Also, note that you committed the closing tag </li>. when updating it to a div tag, kindly remember to add the closing </div> tag.